### DISCUSSION TITLE
Serenity 5.x - theming question

### DISCUSSION THREAD

**Discussion by kilroyFR** [serenity-is/Serenity#6192] [Help]
Hey All, i am currently in the process of planning migration of my current Serenity .net4 version to .netCore (so gonna need again to register for startsharp also).
Just discovered today that there is some new theme. 

My question is quite simple : 
* is it possible to use the "old" AdminLTE theme (cannot find it in the demo app) as my customers are tied to this ergonomy and i dont think i will be able to force them to switch to new theme design.

LMK please. Happy to get back to .netCore5 version. Thanks

**Comment by volkanceylan** [comment]
You may continue using old css files, but we will not support it officially.

The new theme is based on BS5 and old one is legacy and heavy. But if you are only talking about having a header like before instead of a sidebar band (for login, theme change etc.) we may offer a classic theme option, but it may not match the old theme exactly.

**Comment by volkanceylan** [comment]
https://github.com/serenity-is/Serenity/issues/6210
